src/Workspaces/Core/Portable/Workspace/Host/WorkspaceEventListener/IWorkspaceEventListener.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    /// <summary>
+    /// provide a way for features to lazily subscribe to workspace events
+    /// </summary>
+    internal interface IWorkspaceEventListener
+    {
+        void Listen(Workspace workspace);
```

Comment:
I admit I wish this called more StartListening/StopListening so the pairwise thing is a bit clearer.

Reply:
sure.